src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -429,51 +972,20 @@ tests:
-          CHIP:DMG: 	[
-          CHIP:DMG: 		AttributeReportIB =
-          CHIP:DMG: 		{
-          CHIP:DMG: 			AttributeDataIB =
-          CHIP:DMG: 			{
-          CHIP:DMG: 				DataVersion = 0x92ecb45d,
-          CHIP:DMG: 				AttributePathIB =
-          CHIP:DMG: 				{
-          CHIP:DMG: 					Endpoint = 0x1,
-          CHIP:DMG: 					Cluster = 0x62,
-          CHIP:DMG: 					Attribute = 0x0000_0002,
-          CHIP:DMG: 				}
-          CHIP:DMG:
-          CHIP:DMG: 				Data = [
-          CHIP:DMG:
-          CHIP:DMG: 					{
-          CHIP:DMG: 						0x0 = 4,
-          CHIP:DMG: 						0x1 = 4,
-          CHIP:DMG: 						0x2 = 0,
-          CHIP:DMG: 						0x3 = false,
-          CHIP:DMG: 						0x4 = 3,
-          CHIP:DMG: 						0xfe = 1,
-          CHIP:DMG: 					},
-          CHIP:DMG: 				],
-          CHIP:DMG: 			},
-          CHIP:DMG:
-          CHIP:DMG: 		},
-          CHIP:DMG:
-          CHIP:DMG: 	],
-          CHIP:DMG:
-          CHIP:DMG: 	InteractionModelRevision = 11
-          CHIP:DMG: }
-          CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Attribute 0x0000_0002 DataVersion: 2464986205
-          CHIP:TOO:   FabricSceneInfo: 1 entries
-          CHIP:TOO:     [1]: {
-          CHIP:TOO:       SceneCount: 4
-          CHIP:TOO:       CurrentScene: 4
-          CHIP:TOO:       CurrentGroup: 0
-          CHIP:TOO:       SceneValid: FALSE
-          CHIP:TOO:       RemainingCapacity: 7
-          CHIP:TOO:       FabricIndex: 1
-          CHIP:TOO:      }
-          CHIP:DMG: Refresh LivenessCheckTime for 9224 milliseconds with SubscriptionId = 0x7049d607 Peer = 01:0000000000000001
+          ./chip-tool scenesmanagement subscribe fabric-scene-info 100 200 2 1 --commissioner-name beta
+
+          Verify that the DUT sends a report data to TH2 for FabricSceneInfo after the MinIntervalFloor time; store the RemainingCapacity field from this fabric’s entry reported in FabricSceneInfo into Remaining2ndCapacity; verify Remaining2ndCapacity equals (MaxRemainingCapacity) on TH2(Chip-tool) log and below is the sample log provided for the raspi platform:
+
+          [1705915866.717469][21433:21435] CHIP:DMG: }
+          [1705915866.717543][21433:21435] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0062 Attribute 0x0000_0002 DataVersion: 2122634913
+          [1705915866.717578][21433:21435] CHIP:TOO:   FabricSceneInfo: 1 entries
+          [1705915866.717600][21433:21435] CHIP:TOO:     [1]: {
+          [1705915866.717607][21433:21435] CHIP:TOO:       SceneCount: 0
+          [1705915866.717613][21433:21435] CHIP:TOO:       CurrentScene: 0
+          [1705915866.717619][21433:21435] CHIP:TOO:       CurrentGroup: 0
+          [1705915866.717624][21433:21435] CHIP:TOO:       SceneValid: FALSE
+          [1705915866.717630][21433:21435] CHIP:TOO:       RemainingCapacity: 7
+          [1705915866.717635][21433:21435] CHIP:TOO:       FabricIndex: 2
+          [1705915866.717640][21433:21435] CHIP:TOO:      }
+
       disabled: true
```

Comment:
Is there a reason why TH3 is never removed in the tesplace spec? It seems like we just leave the device with those resources occupied.

This is likely a follow up if my understanding is correct that it should be removed

Reply:
Followup here https://github.com/project-chip/connectedhomeip/issues/32127